src/views/TeamCreationView.vue

```diff
@@ -112,6 +112,7 @@ export default {
       }).then((response) => {
         // eslint-disable-next-line
         console.log(response);
+        this.$store.dispatch('getAllTeams');
```

Comment:
What's this line for?

Reply:
That line updates the list of teams in the vuex store, so that when you go from the individual team page to the available-teams page without reloading the team is there